.php-cs-fixer.dist.php

```diff
@@ -43,9 +43,9 @@
     ->setParallelConfig(ParallelConfigFactory::detect()) // @TODO 4.0 no need to call this manually
     ->setUnsupportedPhpVersionAllowed(true)
     ->setRiskyAllowed(true)
-    ->registerCustomFixers([
-        new ConfigurableFixerTemplateFixer(),
-    ])
+    ->registerCustomFixers(class_exists(ConfigurableFixerTemplateFixer::class) ? [
```

Comment:
not so pretty yet, will get prettier with improvements on registering custom sets/rules